inst/template/index.html

```diff
@@ -6,12 +6,93 @@
+                  </svg>
+                </div>
+                <span class="nav-text">
+                  Configure Tableau to use your analytics extension
+                </span>
+              </a>
+            </li>
+            <li>
+              <a href = "./__docs__/">
+                <div class="menuitem">
+                  <!-- INFO CIRCLE SGV -->
+                  <svg 
+                    aria-hidden="true" 
+                    role="img" 
+                    viewBox="0 0 512 512" 
+                    style="height:1em;width:1em;vertical-align:-0.125em;margin-left:auto;margin-right:auto;font-size:inherit;fill:currentColor;overflow:visible;position:relative;"
+                    >
+                      <path d="M256 8C119.043 8 8 119.083 8 256c0 136.997 111.043 248 248 248s248-111.003 248-248C504 119.083 392.957 8 256 8zm0 110c23.196 0 42 18.804 42 42s-18.804 42-42 42-42-18.804-42-42 18.804-42 42-42zm56 254c0 6.627-5.373 12-12 12h-88c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h12v-64h-12c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h64c6.627 0 12 5.373 12 12v100h12c6.627 0 12 5.373 12 12v24z"></path>
+                  </svg>
+                </div>
+                <span class="nav-text">
+                  View your extension's Open API documentation
+                </span>
+              </a>
+            </li>
+            <li>
+              <a href = "./help">
+                <div class="menuitem">
+                  <!-- QUESTION CIRCLE SGV -->
+                  <svg 
+                    aria-hidden="true" 
+                    role="img" 
+                    viewBox="0 0 512 512" 
+                    style="height:1em;width:1em;vertical-align:-0.125em;margin-left:auto;margin-right:auto;font-size:inherit;fill:currentColor;overflow:visible;position:relative;"
+                  >
+                    <path d="M504 256c0 136.997-111.043 248-248 248S8 392.997 8 256C8 119.083 119.043 8 256 8s248 111.083 248 248zM262.655 90c-54.497 0-89.255 22.957-116.549 63.758-3.536 5.286-2.353 12.415 2.715 16.258l34.699 26.31c5.205 3.947 12.621 3.008 16.665-2.122 17.864-22.658 30.113-35.797 57.303-35.797 20.429 0 45.698 13.148 45.698 32.958 0 14.976-12.363 22.667-32.534 33.976C247.128 238.528 216 254.941 216 296v4c0 6.627 5.373 12 12 12h56c6.627 0 12-5.373 12-12v-1.333c0-28.462 83.186-29.647 83.186-106.667 0-58.002-60.165-102-116.531-102zM256 338c-25.365 0-46 20.635-46 46 0 25.364 20.635 46 46 46s46-20.636 46-46c0-25.365-20.635-46-46-46z"></path>
+                  </svg>
+                </div>
+                <span class="nav-text">
+                  Help with plumbertableau and RStudio Connect
```

Comment:
```suggestion
                  Read up on plumbertableau, Tableau, and RStudio Connect
```
Changed this heading to match syntactically with the others. "Read up on" is a little informal, but I like the sound of it. Better than other things I considered, like "Get help with" (which suggests you might be able to ask someone questions) or "Read more about" (which sounds generic and marketing-y).